setup.py

```diff
@@ -221,27 +221,43 @@ def run(self):
         ''' run command '''
 
         has_errors = False
+        playbooks = set()
+        included_playbooks = set()
 
         for yaml_file in find_files(
                 os.path.join(os.getcwd(), 'playbooks', 'byo'),
                 None, None, r'\.ya?ml$'):
             with open(yaml_file, 'r') as contents:
-                for line in contents:
-                    # initialize_groups.yml is used to identify entry point playbooks
-                    if re.search(r'initialize_groups\.yml', line):
-                        print('-' * 60)
-                        print('Syntax checking playbook: %s' % yaml_file)
-                        try:
-                            subprocess.check_output(
-                                ['ansible-playbook', '-i localhost,',
-                                 '--syntax-check', yaml_file]
-                            )
-                        except subprocess.CalledProcessError as cpe:
-                            print('{}Execution failed: {}{}'.format(
-                                self.FAIL, cpe, self.ENDC))
-                            has_errors = True
-                        # Break for loop, no need to continue looping lines
-                        break
+                for task in yaml.safe_load(contents.read()):
```

Comment:
nit: I realize this wasn't something added in this PR but technically ``contents`` is a file instance. It reads a bit off to me to need to ``read`` the ``contents`` to get the contents of the instance.